doc/lua_api.txt

```diff
@@ -3440,6 +3452,15 @@ Definition tables
         light_source = 0, -- Amount of light emitted by node
         damage_per_second = 0, -- If player is inside node, this damage is caused
         node_box = {type="regular"}, -- See "Node boxes"
+        connects_to = nodenames, --[[
+        * Used for nodebox nodes with the type == "connected"
+        * Specifies to what neighboring nodes connections will be drawn
+        * e.g. `{"group:fence", "default:wood"}` or `"default:stone"` ]]
+        connect_sides = { "up", "down", "east", "f", "s" }, --[[
+        * tells connected nodebox nodes to connect only to these sides of this node,
+        * valid entries: "u" = top of node, "d" = bottom, "n" | "b" = back side,
+        * "s" | "f" = front side, "e" | "l" = left side, "w" | "r" = right side.
+        * additional characters are ignored. ]]
```

Comment:
It prints a warning:

https://github.com/sofar/minetest/commit/93a5d91f634e6692b933fba5a2e81583aaf04054#diff-fc5bd7f599797bce163af517e57a314cR574